src/Microsoft.DotNet.Cli.Utils/Command.cs

```diff
@@ -88,14 +88,8 @@ private static string ResolveExecutableFromPath(string executable, ref string ar
             if (useCmd)
             {
                 var comSpec = Environment.GetEnvironmentVariable("ComSpec");
-
-                // cmd doesn't like "foo.exe ", so we need to ensure that if
-                // args is empty, we just run "foo.exe"
-                if (!string.IsNullOrEmpty(args))
-                {
-                    executable = (executable + " " + args).Replace("\"", "\\\"");
-                }
-                args = $"/C \"{executable}\"";
+                // wrap 'executable' within quotes to deal woth space in its path.
```

Comment:
Also, isn't the comment above true anymore? If you have a command foo.exe without args, won't you end up with ""foo.exe" "? Is this valid?